### PR TITLE
Ozerbt code review

### PR DESCRIPTION
**Pull request by ozerbt**
PuzzlePic is an application that receives images from the user's personal storage or from an online database and shuffles it into a puzzle that can be solved.  The application was available on UC GitHub in time and there was enough documentation for the code to be understandable.  I ran the program and it compiled correctly with no errors. 

I made three UI commits in my code review:

- I went ahead and made the buttons a bit smaller as they were abnormally large and took up a good amount of screen space. 
- I changed the background color of the "create new puzzle" button so its more apparent to the user that these two options are clickable. 
- I replaced the background keyword with backgroundTint as the original was preventing default on click highlight colors to be displayed. 

Overall, I learned optimal button standards for android screens (https://support.google.com/accessibility/android/answer/7101858?hl=en)
and I learned that the background keyword prevents certain default features such as on click or on hold events in android studio. 
(https://www.youtube.com/watch?v=xWWnrh_Gks0)

3 commits of mine: 
0a867d5
80e5aec
ae35188

### CODE REVIEW

File: app/src/main/res/layout/main_fragment.xml [hunk 1]
```diff
@@ -13,20 +13,20 @@
...
-        android:visibility="visible"
-        android:layout_alignParentBottom="true"
         android:elevation="2dp"
+        android:visibility="invisible"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        tools:visibility="visible" />
+        tools:visibility="invisible" />
```
Comment (discospiff) [inline app/src/main/res/layout/main_fragment.xml:29]: This looks like new functionality.

File: app/src/main/res/layout/main_fragment.xml [hunk 2]
```diff
@@ -326,7 +326,7 @@
             android:text="@string/navbar_create_puzzle_button"
             android:textAppearance="@style/TextAppearance.AppCompat.Large"
             android:textColor="@color/customBlack"
-            app:backgroundTint="@color/creamBackground" />
+            app:backgroundTint="#D8CDAC86" />
```
Comment (discospiff) [inline app/src/main/res/layout/main_fragment.xml:329]: Keep this as it were.  Better to use colors.xml than to specify a color in the layout XML.

**Comment by discospiff** [comment]
There are several UI changes, but I'm not entirely clear on how they reduce technical debt of the source code.  Thus, I'd consider this functionality more than a code review, and would hold off on merging, unless the product owner likes the functionality changes.

**Comment by erswagner7** [comment]
Decided not to use code review changes.

**Pull request closed without merging**